graph-refresh/overlays/test/cronjob.yaml

```diff
@@ -61,21 +47,21 @@ spec:
                     configMapKeyRef:
                       key: rsyslog-port
                       name: thoth
-                - name: PROMETHEUS_PUSHGATEWAY_URL
-                  valueFrom:
-                    configMapKeyRef:
-                      name: prometheus
-                      key: pushgateway-url
                 - name: SENTRY_DSN
                   valueFrom:
                     secretKeyRef:
                       name: thoth
                       key: sentry-dsn
-                - name: KNOWLEDGE_GRAPH_HOST
+                - name: THOTH_DEPLOYMENT_NAME
```

Comment:
is this being used?

Reply:
Sorry I updated the file can you please check?